spring-test-dbunit-sample/src/test/java/com/github/springtestdbunit/sample/service/PersonServiceTest.java

```diff
@@ -18,7 +18,7 @@
 import com.github.springtestdbunit.sample.entity.Person;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration
+@ContextConfiguration({"classpath:com/github/springtestdbunit/sample/service/applicationContext.xml"})
```

Comment:
Why does the existing configuration need to be changed?
```suggestion
@ContextConfiguration
```